src/components/paragraphs/stanford-banner/banner-paragraph.tsx

```diff
@@ -60,12 +60,12 @@ const BannerParagraph = ({paragraph, eagerLoadImage, ...props}: Props) => {
             }
 
             {paragraph.suBannerSupHeader &&
-              <div className="order-1 font-semibold">
+              <div className="order-1 text-09em font-semibold">
                 {paragraph.suBannerSupHeader}
               </div>
             }
 
-            <Wysiwyg html={paragraph.suBannerBody?.processed} className="order-3"/>
+            <Wysiwyg html={paragraph.suBannerBody?.processed} className="order-3 text-m0"/>
```

Comment:
There is a 100ch width applied to the p within this. It keeps it from aligning flush right to the image edge. @pookmish Do you know the best way to fix that?

Reply:
I'll fix in another commit

Reply:
I'm not sure I know what you mean. when should the text align to the image edge of the banner?

Reply:
I'm sorry @pookmish That comment should have gone on the media with caption file.

https://github.com/pookmish/cardinal-sites-next/blob/a440083e5492da15e08caa664c6c2f3ec093bc46/src/components/paragraphs/stanford-media-caption/media-caption-paragraph.tsx#L40

Reply:
Ok, but from what I see, it is aligning to the right of the image. I'm not sure what you are seeing different 
![image](https://github.com/pookmish/cardinal-sites-next/assets/7185045/c50f7a3e-9bf8-4f19-86ab-ac7840ba9e08)

Reply:
Well, it is right aligned fine now for me too.  🤷🏼‍♀️